src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -135,6 +174,27 @@ void initIoDrivetrain(void)
                        drivetrain_unit_back_left, drivetrain_unit_back_right);
 }
 
+void initIoNetworking()
+{
+    // TODO this needs to be hooked up to the channel dial on the robot, when available
+    unsigned channel = 0;
+
+    io_proto_multicast_communication_init();
+
+    primitive_msg_listener_profile = io_proto_multicast_communication_profile_create(
+        "primitive_msg_listener_profile", MULTICAST_CHANNELS[channel], PRIMITIVE_PORT,
+        &primitive_msg, PrimitiveMsg_fields, 150);
```

Comment:
How did we come up with `150`? Can it be made a constant somewhere?

Reply:
this one is tough, we don't know the size of the msg before hand, so we are just giving a rough ballpark on how big the message can be, I can set them all to 1000 and define a constant somewhere, but generally not sure what the best way to go about it is

Reply:
I think just assuming they'll all be some very large value (1000 should be fine for now) is probably the safest bet.

Reply:
at least making it a named constant with a comment would be much better, even if the number itself is somewhat arbitrary.

Reply:
bumping ^

Reply:
done